packages/sdk/src/conversation/command.ts

```diff
@@ -40,11 +63,43 @@ export class CommandBot {
   /**
    * Registers commands into the command bot.
    *
-   * @param commands The command to registered.
+   * @param commands The commands to register.
    */
   public registerCommands(commands: TeamsFxBotCommandHandler[]): void {
     if (commands) {
       this.middleware.commandHandlers.push(...commands);
     }
   }
+
+  /**
+   * Registers a sso command into the command bot.
+   *
+   * @param command The command to register.
+   */
+  public registerSsoCommand(ssoCommand: TeamsFxBotSsoCommandHandler): void {
+    if (ssoCommand) {
+      if (!this.middleware.getActivityHandler()) {
+        this.middleware.setActivityHandler(new DefaultSsoExecutionActivityHandler(this.ssoConfig));
+      }
+      this.middleware.commandHandlers.push(ssoCommand);
+      this.middleware.activityHandler?.addCommand(ssoCommand);
+    }
```

Comment:
should we throw an error if ssoCommand is empty? there may be no case to pass undefined to this function, so if we found ssoCommand is undefined, there may be some user code error

Reply:
OK, thanks, will update

Reply:
Seems typescript can check whether the parameter is invalid, so I remove the check in function body